v3/util/setup-credentials.sh

```diff
@@ -34,6 +34,13 @@ if [ -f ${HOMEDIR}/.ssh/id_rsa ]; then
     cat ${HOMEDIR}/.ssh/id_rsa.pub >> ${HOMEDIR}/.ssh/authorized_keys
 fi
 
+# Save the RDS password to environment variable in control tier
+# TODO: :(
+if [[ "$NODE_ROLE" = "control" && $AV_SECRETS == *"RDSPASSWORD"* ]]; then
+    RDSPASSWORD=`sudo docker run behance/docker-aws-secrets-downloader --table $TABLE --key secrets --name RDSPASSWORD --format plain`
+    sudo echo "RDSPASSWORD=$RDSPASSWORD" >> /etc/environment
```

Comment:
@matthewdfuller so this doesn't work unless the bootstrap script is run as `sudo`. Not sure how we want to handle this.